modules/azure/aks-global/log-analytics-workspace.tf

```diff
@@ -0,0 +1,20 @@
+resource "azurerm_log_analytics_workspace" "aks" {
+  name                = "aks-${var.environment}-${var.location_short}-${var.name}-${var.unique_suffix}"
+  location            = data.azurerm_resource_group.this.location
+  resource_group_name = data.azurerm_resource_group.this.name
+  sku                 = "PerGB2018"
+  retention_in_days   = 30
+}
+
+resource "azurerm_log_analytics_solution" "logs" {
```

Comment:
This is a pre-built solution that Azure provides: https://docs.microsoft.com/en-us/azure/azure-monitor/containers/container-insights-overview

Reply:
Removed this since we don't gather any vm info.